site/content/docs/5.2/components/pagination.md

```diff
@@ -121,12 +121,72 @@ Or with `.justify-content-end`:
 <!-- Boosted mod -->
 ## Responsive behavior
 
-To avoid pagination wrapping or overflowing, we limit the number or pagination item to display depending on breakpoint.
+{{< callout warning >}}
+**Heads up—our responsive pagination is experimental and opt-in as of v5.2.1!** We included it in our documentation's CSS to demonstrate it for you, but it's disabled by default. Keep reading to learn how to enable it in your projects.
+{{< /callout >}}
+
+By default, every item is visible. Here is an example **without** our responsive behavior.
 
 {{< example >}}
```

Comment:
We shouldn't have an example here showing 13 elements since the users should display a maximum of 10 elements (if I remember well what's described in the DSM). If they have use cases with more, they should handle it themselves or use the smart responsive behavior.